common/src/main/java/com/wynntils/core/consumers/overlays/OverlayManager.java

```diff
@@ -205,7 +205,10 @@ private void recreateGroupOverlaysWithIds(OverlayGroupHolder holder, List<Intege
     // region Ticking
     @SubscribeEvent
     public void onTick(TickEvent event) {
-        enabledOverlays.forEach(Overlay::tick);
+        enabledOverlays.forEach(overlay -> {
+            overlay.tick();
+            overlay.updateCache();
```

Comment:
This is reallyyyy not supposed to be updated on every tick. It just needs to be done in `updateConfigOption`. The only reason your game could still run with this is because function manager already does caching on function evaluation.

Reply:
Yes, my bad.